src/ert/job_runner/job.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, job_data, index, sleep_interval=1):
         self.std_out = job_data.get("stdout")
 
     def run(self):
+        # pylint: disable=consider-using-with
```

Comment:
What do you think about commenting `# stdin, stout and stderr are closed on the bottom of the function`?

Reply:
I just got this idea from your other PR, but it is a minor thing not needed if you prefer so.